### PR TITLE
Add wifi and fix touchscreen on castor

### PR DESCRIPTION
**Pull request by opendata26**
also adds xwayland module

Dont merge this until the upcoming firmware-adreno package is merged

### CODE REVIEW

File: aports/device/device-sony-castor-windy/device-sony-castor_windy.post-install [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Enable networking service (requires /etc/interfaces, which is configured below)
+rc-update add networking default
```
Comment (ollieparanoid) [inline aports/device/device-sony-castor-windy/device-sony-castor_windy.post-install]: Heads up: that might break when we switch to NetworkManager (see #388 for discussion).

**Comment by opendata26** [comment]
I haven't tried the proprietary driver, i meant that is possible.
The firmware in the repo is needed for freedreno to work .

**Comment by ollieparanoid** [comment]
Okay, I've mixed that up! Then this PR is ready from my point of view, as soon as the `firmware-adreno` package is.

**Comment by opendata26** [comment]
Done

**Pull request merged**